app/Http/Middleware/CheckLimitations.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Support\Facades\Auth;
+
+class CheckLimitations
```

Comment:
I would just change this because it's not super clear what it does.

```suggestion
class CheckAccountLimitations
```